tests/testing/test_dispatch.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+import pytest
+
+from tchannel import TChannel, Request
+from tchannel.errors import BadRequestError
+from tchannel.testing.dispatch import handle
+
+
+@pytest.fixture
+def server():
+    tchannel = TChannel('test_server')
+
+    @tchannel.raw.register
+    def hello(request):
+        return request.body + " world"
+
+    return tchannel
+
+
+@pytest.mark.gen_test
+def test_handle_success(server):
+    request = Request(
+        body="hello",
+        endpoint="hello",
+    )
+    response = yield handle(server, request)
```

Comment:
that shouldn't in this diff. It is different thing. I don't know if people want's it or not, because if they call the endpoint method directly, the return value from the method is not a tchannel response object.